os/logm/Kconfig

```diff
@@ -19,33 +19,33 @@ config SYSLOG2LOGM
 	bool "Route all syslogs through logm"
 	default y
 
-config LOGM_BUFFER_COUNT
-	int "Number of buffers to store log messages"
-	default 80
+config LOGM_TIMESTAMP
+	bool "Prepend timestamp to message"
+	default n
+
+config LOGM_BUFFER_SIZE
+	int "Logm Buffer size"
+	default 10240
```

Comment:
When executing 'logm' in TASH shell, the buffer size is 5120 (bytes). Is it correct?